### PR TITLE
Update uno from uno/master

### PR DESCRIPTION
**Pull request by mortend**
This updates uno to current master.

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Experimental.TextureLoader/CIL/Uno/BufferAccessor.uno [hunk 1]
```diff
@@ -2,9 +2,12 @@ using Uno.Compiler.ExportTargetInterop;
 
 namespace Uno
 {
-	[DotNetType("Uno.BufferAccessor")]
 	extern(DOTNET) public static class BufferAccessor
 	{
-		public extern static byte[] GetBytes(Uno.Buffer buffer);
+		[Obsolete]
+		public static byte[] GetBytes(Uno.Buffer buffer)
```
Comment (kusma) [inline Source/Experimental.TextureLoader/CIL/Uno/BufferAccessor.uno]: This needs a changelog-entry, as it is public API.

File: Source/Experimental.TextureLoader/CIL/Uno/Texture.uno [hunk 2]
```diff
@@ -1,153 +0,0 @@
...
-		D_32_Float = 500,
-		DSX_32_8_24_Float_UInt_Typeless = 501,
-		DS_24_8_UInt_Normalize_UInt = 502,
-
-		D_16_UNorm = 503,
-		R_16_UNorm = 504
-	}
-
-	[DotNetType("Uno.Native.Textures.Texture")]
-	extern(DOTNET) public class Texture : Uno.IDisposable
```
Comment (kusma) [inline Source/Experimental.TextureLoader/CIL/Uno/Texture.uno]: Same here, this needs to be mentioned in the changelog. Ideally, this should have been marked as deprecated in the previous release, though.

Comment (kusma) [reply Source/Experimental.TextureLoader/CIL/Uno/Texture.uno]: How can we remove this, when this class is in use in CilTextureLoader?

File: Source/Experimental.TextureLoader/CIL/Uno/BufferAccessor.uno (hunk 1 above)
Comment (mortend) [reply Source/Experimental.TextureLoader/CIL/Uno/BufferAccessor.uno]: Uno Changelog: https://github.com/fusetools/uno/commit/589ba675225a0bdf6aa554ea86efd1473b65fa07

**Comment by mortend** [comment]
Replaced by #620

**Pull request closed without merging**